doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -757,11 +757,6 @@ the toplevel, and using them in source files is discouraged.
    :n:`-R @string @dirpath`. It adds the physical directory string and all its
    subdirectories to the current |Coq| loadpath.
 
-   .. cmdv:: Add Rec LoadPath @string
```

Comment:
You have removed this variant but not the `Add LoadPath @string` variant. Is this a mistake?

Reply:
Ups, I was under the impression I had effectively taken care of both of them.